WordPress/Classes/ViewRelated/Gutenberg/GutenbergImageLoader.swift

```diff
@@ -13,11 +13,11 @@ class GutenbergImageLoader: NSObject, RCTImageURLLoader {
         self.post = post
     }
 
-    func canLoadImageURL(_ requestURL: URL!) -> Bool {
-        return !requestURL.isFileURL
+    func canLoadImageURL(_ requestURL: URL) -> Bool {
+        return requestURL.isFileURL
```

Comment:
This seems to change how this method works, not just its signature – do we want that?

Reply:
+1 on this question.

The docs for `isFileURL` say: Returns true if the scheme is `file:`.

Reply:
Sorry this was a mistake after seeing so many IUOs.  I'm glad I had your eyes on this review!

Reply:
Rolled back this change.